File: 백준/Bronze/2592. 대표값/대표값.c
#include <stdio.h>

int main(void) {

  int result = 0, max = -1, max_index;
  int arr[100] = {0};
  for(int i = 0; i < 10; i++){
    int a;
    scanf("%d", &a);

    result += a;
    arr[(a / 10) - 1]++;
  }

  for(int i = 0; i < 100; i++){
    if(arr[i] > max){
      max = arr[i];
      max_index = i;
    }
  }

  printf("%d\n%d", result / 10, (max_index + 1) * 10);
  
  return 0;
}

File: 백준/Bronze/2592. 대표값/대표값.py
arr = [0] * 100
result, max = 0, -1
max_index = 0
for _ in range(10):
    a = int(input())
    
    result += a
    arr[(a // 10) - 1] += 1


for i in range(100):
    if arr[i] > max:
        max = arr[i]
        max_index = i


print(result//10)
print((max_index + 1) * 10)